network/dns_frame.ksy

```diff
@@ -0,0 +1,121 @@
+meta:
+  id: dns_packet
+  title: DNS (No support for Auth-Name + Add-Name for simplicity)
+  application: https://www.ietf.org/rfc/rfc1035.txt
+  endian: be
+seq:
+  - id: transaction_id
+    doc: "ID to keep track of request/responces"
+    type: u2
+  - id: flags
+    type: u2
+    doc: "Please add support for bit-flags!"
+    enum: flags_type
+  - id: questions
```

Comment:
RFC refers to these fields as QDCOUNT, ANCOUNT, NSCOUNT, ARCOUNT, or "question", "answer", "authority records", "additional records" sections. I'm not sure about reusing these names, but at the very least we should use stable naming scheme everywhere and refer to the same stuff repeatedly using the same identifiers.